src/fmeval/eval_algorithms/classification_accuracy.py

```diff
@@ -274,14 +297,28 @@ def _generate_columns(row: Dict[str, Any]) -> Dict[str, Any]:  # pragma: no cove
 
     def _get_score(self, y_true, y_pred, eval_fn: Callable[..., float]) -> float:
```

Comment:
Please add unit tests for `_get_score` for the various cases (binary recall, binary precision, multiclass recall, multiclass precision, etc).